src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -1,11 +1,25 @@
 import { Breadcrumb, BreadcrumbItem, BreadcrumbLink } from '@chakra-ui/react'
+import { useMemo } from 'react'
 import type { BreadcrumbsRoute } from 'react-router-breadcrumbs-hoc'
 import withBreadcrumbs from 'react-router-breadcrumbs-hoc'
+import type { RouteComponentProps } from 'react-router-dom'
 import { Link } from 'react-router-dom'
 import { AccountLabel } from 'components/AssetHeader/AccountLabel'
+import { Text } from 'components/Text/Text'
 import { selectAssetById } from 'state/slices/selectors'
 import { useAppSelector } from 'state/store'
 
+const GetTranslatedPathPart = ({ match }: { match: RouteComponentProps['match'] }) => {
+  // a hack to gain the ability to translate the breadcrumbs
+  // takes out the path part from the url
+  // and uses navbar.[pathPart] for the translation
+  const pathFragments = useMemo(() => match.url.split('/'), [match.url])
+  // getting the last part from the url, since this works recursively
+  const pathPart = pathFragments[pathFragments.length - 1]
+  // translates `/` to home
+  return <Text translation={`navBar.${pathPart ? pathPart : 'home'}`} />
+}
+
 const GetAccountName = (props: any) => {
```

Comment:
```suggestion
const getAccountName = (props: any) => {
```

while we're in here

Reply:
@0xdef1cafe These are actually React functional components and their name should start with Uppercase letters. this one actually seems like it can be changed to lowercase, but I think keeping names casing consistent is better here.